src/components/Settings.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import PropTypes from "prop-types";
+import FA from "@fortawesome/react-fontawesome";
+import { Box, Button } from "@hackclub/design-system";
+import Slider from "components/Slider";
+
+const Settings = ({
+    onRadiusChange,
+    onSystemChange,
+    onViewChange,
+    searchRadius,
+    showAllClubs,
+    useImperialSystem,
+    ...props
+}) => (
+    <Box {...props}>
+        {!showAllClubs && (
+            <Slider
+                defaultValue={searchRadius}
+                min={1}
+                max={100}
+                onChange={onRadiusChange}
+            />
+        )}
+        <Button.button onClick={onSystemChange} inverted={!useImperialSystem} disabled={showAllClubs} ml={2}>
```

Comment:
remove the `ml` on line 28, switch to `mx` here

Reply:
What's the reason? This change adds extra space to the right.